gitlab/docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+
```

By using the `app_proxy` we can avoid exposing the UI port. You can use the proxy like this:
```suggestion
  app_proxy:
    environment:
      APP_HOST: gitlab_gitlab_1
      APP_PORT: 8929
      PROXY_AUTH_ADD: "false"
      
```
Note: since GitLab is offering its own authentication solution, we don't need to use the one provided by the proxy, therefore we can skip it with `PROXY_AUTH_ADD: "false"`